dvc/repo/__init__.py

```diff
@@ -68,11 +68,9 @@ def __init__(self, root_dir=None):
 
         core = self.config.config[Config.SECTION_CORE]
 
-        logger.setLevel(
-            logging.getLevelName(
-                core.get(Config.SECTION_CORE_LOGLEVEL, "info").upper()
-            )
-        )
+        level = core.get(Config.SECTION_CORE_LOGLEVEL)
+        if level:
+            logger.setLevel(level.upper())
```

Comment:
The idea of log level defined in repo config looks a bit ill-defined. What if we have instantiated several Repos? The one instantiated later will just override everything. The order will be meaningful, which feels weird.

Reply:
Yes, that probably should be changed. But do you think it is a scope of current issue? Or should we rather introduce this change and make new task for removing the config from Repo?

Reply:
@Suor , @pared , this could be solved on a separate PR, maybe opening an issue for discussion.
This PR is kind of important, otherwise we would have an unusable `--quiet` option.